server/db/index.js

```diff
@@ -1,57 +1,93 @@
-var Shepherd = orm.define('Shepherd', {
-  user_id: Sequelize.INTEGER,
-  event_id: Sequelize.INTEGER
-});
-
-var Sheep = orm.define('Sheep', {
-  user_id: Sequelize.INTEGER,
-  event_id: Sequelize.INTEGER
-});
-
-var Event = orm.define('Event', {
-  eventName: Sequelize.STRING,
-  start: Sequelize.DATE,
-  end: Sequelize.DATE,
-  location: Sequelize.STRING,
-  minParticipants: Sequelize.INTEGER,
-  maxParticipants: Sequelize.INTEGER,
-  action: Sequelize.STRING
-});
-
-var Observation = orm.define('Observation', {
-  user_id: Sequelize.INTEGER,
-  event_id: Sequelize.INTEGER,
-  content: Sequelize.STRING,
-  completed: Sequelize.BOOLEAN
-});
-
-// table relations
-// TODO: build out table relations
-
-Shepherd.hasMany(Event);
-User.hasMany(Observation);
-Event.hasMany(Observation);
-
-// sync all tables
-User.sync();
-Shepherd.sync();
-Sheep.sync();
-Event.sync();
-Observation.sync();
-
-// export all tables
+var config = require('../config/environment');
+
+var User, Event, Observation;
+
+var init = function() {
+  var orm;
+  if (config.postgres.uri) {
+    orm = new Sequelize(config.postgres.uri, config.postgres.options);
+  } else {
+    orm = new Sequelize(config.postgres.database, config.postgres.user, config.postgres.password, config.postgres.options);
+  }
+
+  User = orm.define('User', {
+    username: Sequelize.STRING,
+    facebook_id: Sequelize.STRING
+  });
+
+  Event = orm.define('Event', {
+    eventName: Sequelize.STRING,
+    start: Sequelize.DATE,
+    end: Sequelize.DATE,
+    location: Sequelize.STRING,
+    minParticipants: Sequelize.INTEGER,
+    maxParticipants: Sequelize.INTEGER,
+    action: Sequelize.STRING
+  });
+
+  Observation = orm.define('Observation', {
+    content: Sequelize.STRING,
+    completed: Sequelize.BOOLEAN
+  });
+
+  /**
+   * MODEL RELATIONS
+   * belongsToMany connects a source with multiple targets, and the targets can connect to multiple sources
+   * Using a `through` option will create a new model with foreign keys for the source and target
+   *
+   * Example:
+   * User.belongsToMany(Event, {as: 'ShepherdEvents', through: 'ShepherdEvent'});
+   * Event.belongsToMany(User, {as: 'Shepherds', through: 'ShepherdEvent'});
+   *
+   * We create a many-to-many relationship between users and events, and the join table is called ShepherdEvent.
+   * The ShepherdEvent table will have a foreign key for EventId and UserId.
+   * The User and Event models will have get, set, and add methods. For example, user.getShepherdEvents() 
+   * will return all instances of a ShepherdEvent belonging to that user, and each instance would reference the 
+   * Event the user is a shepherd for.
+   * 
+   * See http://sequelize.readthedocs.org/en/latest/docs/associations/#belongs-to-many-associations
+   *
+   * hasMany connects a source with multiple targets, but the targets can only connect to one source
+   *
+   * See http://sequelize.readthedocs.org/en/latest/docs/associations/#one-to-many-associations
+   */
+
```

Comment:
i think you meant to swap the order. 56 should be shepherds, 57 should be shepherdevents, 58 sheep, 59 sheepevent